src/Xabe.FFmpeg/Conversion/Conversion.cs

```diff
@@ -281,29 +284,79 @@ public IConversion SetOutput(string outputPath)
             OutputFilePath = outputPath;
             _output = $"\"{outputPath}\"";
             return this;
+        }
+        /// <inheritdoc />
+        public IConversion SetVSync(int vsyncmode)

```

Comment:
Change int to enum
![image](https://user-images.githubusercontent.com/13612984/96335541-a4fc0880-1079-11eb-86d9-a51ab6cfb2f3.png)